src/components/shared/MembersList/MembersList.tsx

```diff
@@ -1,48 +1,45 @@
 import React, { ReactNode } from 'react';
 
 import ListGroup, { ListGroupAppearance } from '~shared/ListGroup';
-import { AnyUser, Colony } from '~data/index';
+import { Watcher, Contributor } from '~types';
 
 import MembersListItem from './MembersListItem';
+import { notNull } from '~utils/arrays';
 
-interface Props<U> {
-  colony: Colony;
-  extraItemContent?: (user: U) => ReactNode;
-  onRowClick?: (user: U) => void;
+interface Props {
+  users: (Watcher | Contributor)[];
+  listGroupAppearance?: ListGroupAppearance;
+  canAdministerComments?: boolean;
+  extraItemContent?: (user: (Contributor | Watcher)['user']) => ReactNode;
```

Comment:
If you follow my suggestion from #139 about a dedicated fragment for the `user` field, you could later use it here

Reply:
@jakubcolony I made a change in #139 to have a separate fragment for the User Profile but how does that help here? or were you expecting a different change?